resources/setups/baseline/module_config.json

```diff
@@ -36,5 +36,11 @@
     "ntp": {
       "enabled": true
     }
+  },
+  "dhcp_tests": {
```

Comment:
as per other comments, I'm not sure this should be a top-level construct at the same level as "modules" -- I would have expected "modules" to contain a "dhcp" module...